apps/AppWithWearable/lib/backend/api_requests/stream_api_response.dart

```diff
@@ -110,8 +111,18 @@ void handlePartialResponseContent(String data, Future<dynamic> Function(String)
   }
 }
 
-String qaStreamedBody(String context, List<Message> chatHistory) {
+String qaStreamedBody(dynamic personality, String context, List<Message> chatHistory) {
```

Comment:
For now let's use the description as their personality and reformat the plugins so the description has this personality (ish).